.github/workflows/dependent-repos.yml

```
@@ -0,0 +1,57 @@
+name: Check that repositories submoduling us will still build after this PR
+
+on:
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  android-bindings:
+    runs-on: [self-hosted, Linux, large]
+    container: mobilecoin/builder-install:v0.0.17
+
+    permissions:
+      pull-requests: write
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        path: mobilecoin
+    - name: Check that android-bindings still builds
+      uses: ./mobilecoin/.github/actions/check-dependent-rust-repo
+      with:
+        remoteRepo: mobilecoinofficial/android-bindings
+        submodulePath: mobilecoin
+        id: android-bindings
+        githubToken: ${{ secrets.GITHUB_TOKEN }}
+        buildCmd: cargo build
+
+  full-service:
+    runs-on: [self-hosted, Linux, large]
+    container: mobilecoin/builder-install:v0.0.17
+
+    permissions:
+      pull-requests: write
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        path: mobilecoin
+    - name: Check that full-service still builds
+      uses: ./mobilecoin/.github/actions/check-dependent-rust-repo
+      with:
+        remoteRepo: mobilecoinofficial/full-service
+        submodulePath: mobilecoin
+        id: full-service
+        githubToken: ${{ secrets.GITHUB_TOKEN }}
+        buildCmd: |
+          export CONSENSUS_ENCLAVE_CSS=$PWD/consensus-enclave.css
+          echo "CONSENSUS_ENCLAVE_CSS = $CONSENSUS_ENCLAVE_CSS"
+          if [ ! -f "$CONSENSUS_ENCLAVE_CSS" ]; then
+            CONSENSUS_SIGSTRUCT_URI=$(curl -s https://enclave-distribution.test.mobilecoin.com/production.json | grep consensus-enclave.css | awk '{print $2}' | tr -d \" | tr -d ,)
```

Consider using `jq` instead (confirmed it's present in the `builder-install` image)
```suggestion
            CONSENSUS_SIGSTRUCT_URI=$(curl -s https://enclave-distribution.test.mobilecoin.com/production.json | jq -r .consensus.sigstruct)
```